docs/how-tos/creating-new-events.rst

```diff
@@ -0,0 +1,143 @@
+How to create a new Open edX Event
+==================================
+
+The mechanisms implemented by the Open edX Events library are supported and maintained by the Open edX community.
+Therefore, we've put together a guide on how to add a new event the library so future contributions are effective.
+
+
+1. Propose the new event to the community
```

Comment:
[thoughts] I'm fine with this landing as-is, and iterating as needed, but @ormsbee was thinking more proprietary events might still live in this repo, possibly with a company specific part of the domain. Don't let this be a blocker to landing this PR. This is just a possible discussion for the future.

Reply:
What about this for now:

1. Adding optional to the step
2. Adding at the end:
_This step might be unnecessary for events not facing the Open edX community._

Reply:
1. Maybe land this as-is for now?
2. I wonder if we should create an open github issue and point to it in this doc with something like:
```
.. note::
    There is an open discussion about whether organization scoped events would also exist in openedx-events,
    in which case this step would be optional. See <insert issue link here>.
```

Reply:
That sounds good for now! 

BTW, there's an open issue about private event schemas that might also be related to this conversation. I'll leave it here just in case: https://github.com/openedx/openedx-events/issues/184